src/pages/__test__/RegisterForm.test.js

```diff
@@ -0,0 +1,41 @@
+import { shallow } from 'enzyme';
```

Comment:
I think it would be better if you created a test folder in the forms folder and then put this file there